### PR TITLE
Add tikv-br docs

### PR DESCRIPTION
**Pull request by haojinming**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/11506
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/rawkv-backup-and-restore.md [hunk 1]
```diff
@@ -5,57 +5,8 @@ summary: 了解如何使用 br 命令行工具备份和恢复 RawKV 数据。
 
 # 备份与恢复 RawKV 数据
 
-> **警告：**
->
-> 从 TiDB v6.2.0 开始，RawKV 备份和恢复功能废弃。
+TiKV 可以独立于 TiDB，与 PD 构成 KV 数据库，此时的产品形态为 RawKV。
 
-TiKV 有时可以独立于 TiDB，与 PD 构成 KV 数据库，此时的产品形态为 RawKV。Backup & Restore (BR) 支持对使用 RawKV 的产品进行备份和恢复，本文介绍如何备份和恢复 RawKV。
+TiKV-BR 工具支持对使用 RawKV 的产品进行备份和恢复，也支持将 TiKV 集群中的数据从 `API V1` 备份为 `API V2` 格式， 以实现 TiKV 集群 [`api-version`](https://docs.pingcap.com/zh/tidb/v6.4/tikv-configuration-file#api-version-%E4%BB%8E-v610-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) 的升级。
```
Comment (shichun-0415) [inline br/rawkv-backup-and-restore.md:10]:
```suggestion
TiKV-BR 工具支持对使用 RawKV 的产品进行备份和恢复，也支持将 TiKV 集群中的数据从 `API V1` 备份为 `API V2` 格式， 以实现 TiKV 集群 [`api-version`](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) 的升级。
```

File: br/rawkv-backup-and-restore.md [hunk 2]
```diff
@@ -5,57 +5,8 @@ summary: 了解如何使用 br 命令行工具备份和恢复 RawKV 数据。
...
-    --storage "local://$BACKUP_DIR" \
-    --start 31 \
-    --end 3130303030303030 \
-    --ratelimit 128 \
-    --format hex \
-    --cf default
-```
-
-以上命令会将范围在 `[0x31, 0x3130303030303030)` 的已备份键恢复到 TiKV 集群中。这里键的编码方式和备份时相同。
+更多信息，请参考[TiKV-BR 用户文档]( https://tikv.org/docs/dev/concepts/explore-tikv-features/backup-restore/ )。
```
Comment (shichun-0415) [inline br/rawkv-backup-and-restore.md:12]:
```suggestion
更多信息，请参考 [TiKV-BR 用户文档]( https://tikv.org/docs/dev/concepts/explore-tikv-features/backup-restore/ )。
```

**Review by IANTHEREAL — APPROVED** [review]

File: br/rawkv-backup-and-restore.md [hunk 3]
```diff
@@ -1,61 +1,10 @@
...
-    --storage "local://$BACKUP_DIR" \
-    --start 31 \
-    --end 3130303030303030 \
-    --ratelimit 128 \
-    --format hex \
-    --cf default
-```
-
-以上命令会将范围在 `[0x31, 0x3130303030303030)` 的已备份键恢复到 TiKV 集群中。这里键的编码方式和备份时相同。
+更多信息，请参考 [TiKV-BR 用户文档]( https://tikv.org/docs/latest/concepts/explore-tikv-features/backup-restore-cn/ )。
```
Comment (pingyu) [inline br/rawkv-backup-and-restore.md:10]:
```suggestion
更多信息，请参考 [TiKV-BR 用户文档](https://tikv.org/docs/latest/concepts/explore-tikv-features/backup-restore-cn/)。
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -423,12 +423,12 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
         + 需要同时设置 `storage.enable-ttl = true`。由于 API V2 支持 TTL 特性，因此强制要求打开 `enable-ttl` 以避免这个参数出现歧义。
         + 启用 API V2 后需要在集群中额外部署至少一个 tidb-server 以回收过期数据。该 tidb-server 可同时提供数据库读写服务。可以部署多个 tidb-server 以保证高可用。
         + 需要客户端的支持。请参考对应客户端的 API V2 使用说明。
-        + 从 v6.2.0 版本开始，你可以通过 [RawKV CDC](https://github.com/tikv/migration/tree/main/cdc) 组件实现 RawKV 的 Change Data Capture (CDC)。
+        + 从 v6.2.0 版本开始，你可以通过 [RawKV CDC](https://tikv.org/docs/latest/concepts/explore-tikv-features/cdc/cdc/) 组件实现 RawKV 的 Change Data Capture (CDC)。
```
Comment (pingyu) [inline tikv-configuration-file.md:426]:
```suggestion
        + 从 v6.2.0 版本开始，你可以通过 [RawKV CDC](https://tikv.org/docs/latest/concepts/explore-tikv-features/cdc/cdc-cn/) 组件实现 RawKV 的 Change Data Capture (CDC)。
```

**Review by pingyu — APPROVED** [review]
Rest LGTM~

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]

**Pull request merged**